CONTRIBUTING.md

```diff
@@ -43,8 +43,10 @@ $ git remote add upstream git://github.com/nodejs/node.git
 
 #### Which branch?
 
-For developing new features and bug fixes, the `master` branch should be pulled
-and built upon.
+For developing new features and bug fixes, the `master` branch should be
+pulled and built upon.  See the section on
+[backporting](#backporting-to-current-and-lts-branches) if you would like
+the fix to be backported.
```

Comment:
>if you would like the fix to be backported.

I'm thinking that new contributors probably have no idea if they'd like something to be backported, maybe something like:
`if you think the fix should be applied to current release lines` (or something).